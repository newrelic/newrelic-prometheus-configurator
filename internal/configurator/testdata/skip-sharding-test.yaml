# skip sharding test
sharding:
  kind: hash
  total_shards_count: 2
  shard_index: 1

static_targets:
  jobs:
    - job_name: static-target-job
      skip_sharding: true
      targets:
        - "192.168.3.1:2379"
      metrics_path: /metrics-custom
      extra_relabel_config:
        - source_labels: [ '__name__', 'instance' ]
          regex: node_memory_active_bytes;localhost:9100
          action: drop
      extra_metric_relabel_config:
        - source_labels: [ '__name__', 'instance' ]
          regex: node_memory_active_bytes;localhost:9100
          action: drop

newrelic_remote_write:
  license_key: nrLicenseKey
