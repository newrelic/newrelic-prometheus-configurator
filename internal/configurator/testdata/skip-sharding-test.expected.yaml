# skip-sharding-test.expected
scrape_configs:
  # static_targets
  - job_name: static-target-job
    static_configs:
      - targets:
          - "192.168.3.1:2379"
    metrics_path: /metrics-custom
    relabel_configs:
      - source_labels: [ '__name__', 'instance' ]
        regex: node_memory_active_bytes;localhost:9100
        action: drop
    metric_relabel_configs:
      - source_labels: [ '__name__', 'instance' ]
        regex: node_memory_active_bytes;localhost:9100
        action: drop

remote_write:
  - name: newrelic_rw
    url: https://metric-api.newrelic.com/prometheus/v1/write?collector_name=prometheus-agent
    authorization:
      credentials: nrLicenseKey
